apis/python/src/tiledb/vector_search/ivf_flat_index.py

```diff
@@ -268,6 +303,9 @@ def taskgraph_query(
         from tiledb.vector_search.module import (array_to_matrix, dist_qv,
                                                  partition_ivf_index)
 
+        if resource_class and resources:
```

Comment:
Add a check to make sure that `resources` is not defined for `REALTIME`

Reply:
Sure, done. Originally I just let `tiledb.cloud` throw this error, but you're right, that's inconsistent with how I'm handling other errors, so updated.